dubbo-dependencies-bom/pom.xml

```diff
@@ -363,6 +363,12 @@
                 <groupId>org.jboss.resteasy</groupId>
                 <artifactId>resteasy-jaxrs</artifactId>
                 <version>${resteasy_version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>net.jcip</groupId>
+                        <artifactId>jcip-annotations</artifactId>
```

Comment:
This is no longer needed to be excluded. See discussion on mailing list.

Reply:
I have revert it.